index.bs

```diff
@@ -164,10 +164,10 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
         url: sec-returnifabrupt-shorthands
             text: !
             text: ?
-        text: ECMA-262 §5.2; url: sec-algorithm-conventions
-        text: ECMA-262 §9.1; url: sec-ordinary-object-internal-methods-and-internal-slots
-        text: ECMA-262 §9.3; url: sec-built-in-function-objects
-        text: ECMA-262 §9.4.7; url: sec-immutable-prototype-exotic-objects
+        text: ECMA-262 Algorithm conventions; url: sec-algorithm-conventions
+        text: ECMA-262 Ordinary Object Internal Methods and Internal Slots; url: sec-ordinary-object-internal-methods-and-internal-slots
```

Comment:
Hmm, the capitalization doesn't seem to be fully consistent here. ECMA-262 itself uses Title Case, but we can use Title Case here in the biblio and regular sentence case in the prose itself.

Reply:
Followed your second suggestion.